.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -481,8 +481,6 @@ def write(x=''):
             "default": [
                           "submitty/clang:6.0",
                           "submitty/autograding-default:latest",
-                          "submitty/java:8",
-                          "submitty/java:11",
```

Comment:
How long does it take to download `submitty/java:11`?

Reply:
Not very long at all

Reply:
Let's leave submitty/java:11 in the default list I think for now, given its ubiquitous nature in CS education and the cost of it is minimal. Can definitely kill off java:8 though.

Reply:
No, we are still using java 8 in classes.

Reply:
But our classes are running out of custom containers with jacoco and dafny
submittyrpi/csci2600:java_tools_dafny

So...  I'll not fight about this.